db/sql/sql.go

```diff
@@ -586,6 +610,12 @@ func (tx *Transaction) logQuery(sql string, args ...interface{}) {
 	log.Debug("[%p] Executing SQL query '%s'", tx.transaction, query)
 }
 
+func (tx *CachedTransaction) logQuery(sql string, args ...interface{}) {
```

Comment:
copy-paste of `(tx *Transaction) logQuery`

Reply:
It's hard to avoid as not every transaction.Transaction needs it but maybe will add it there.